data/update.py

```diff
@@ -43,38 +45,65 @@
 # domain-scan information
 SCAN_TARGET = os.path.join(this_dir, "./output/scan")
 SCAN_COMMAND = os.environ.get("DOMAIN_SCAN_PATH", None)
-SCANNERS = os.environ.get("SCANNERS", "inspect,tls,analytics")
-ANALYTICS_URL = "https://analytics.usa.gov/data/live/second-level-domains.csv"
-
-# TODO:
-# --date: override date
-# --skip-scan: skip the scanning part (rely on local scan data)
-# --skip-upload: don't bother uploading anything to S3
-
-def run():
+SCANNERS = os.environ.get("SCANNERS", "inspect,tls,analytics,sslyze")
+ANALYTICS_URL = os.environ.get("ANALYTICS_URL", META["data"]["analytics_url"])
+
+# Options:
+# --date: override date, defaults to contents of meta.json
+# --scan=[skip,download,here]
+#     skip: skip all scanning, assume CSVs are locally cached
+#     download: download scan data from S3
+#     here: run the default full scan
+# --upload: upload scan data and resulting db.json anything to S3
```

Comment:
This adds a very tiny CLI options parser, to support a few options that help distinguish between production, staging, and development use cases for data updating.